packages/logs/src/boot/startLogs.ts

```diff
@@ -35,10 +37,11 @@ export function startLogs(
 
   const reportError = startReportError(lifeCycle)
   const pageExitObservable = createPageExitObservable(configuration)
+  const trackingConsentState = createTrackingConsentState(TrackingConsent.GRANTED)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could we share the same trackingConsentState instance between preStart and start? or maybe just passing the observable to the start?

Reply:
Here are my thoughts: https://github.com/DataDog/browser-sdk/pull/2589#discussion_r1480002807
Let's have a word tomorrow!